File: Haskell/beginner/1075.hs
-- https://judge.beecrowd.com/pt/problems/view/1075
-- Resto 2

-- Main body
main :: IO ()
main = do
    usr_value <- readLn :: IO Int
    mapM_ print [x | x <- [1..10000], x `mod` usr_value == 2]

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1075.c
// https://judge.beecrowd.com/pt/problems/view/1075
// Resto 2

// Libraries
#include <stdio.h>

// Main Function
int main (){
    // Starting variables
    int usr_inputs;

    // Input
    scanf("%d", &usr_inputs);

    // Iteration
    for (int i = 1; i <= 10000; i++){
        if (i % usr_inputs == 2){
            printf("%d\n", i);
        }
    }

    // Ending Code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra